content/influxdb/cloud/write-data/no-code/use-telegraf/dual-write.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
is there any relationship with this content in the "upgrade" procedure (to verify content matches, if need be):
https://docs.influxdata.com/influxdb/cloud/upgrade/v2-to-cloud/#dual-write-to-influxdb-2x-and-influxdb-cloud

Reply:
Yeah this would be the same thing as that step in the upgrade. Could link out to that page possibly?

Reply:
Thanks, @sjwang90, @noramullen1 
Probably best to link the upgrade doc to the actual task of "dual writing"--if both docs are truly referring to the same process. 
Easier to update/maintain